agave_app/AppearanceSettingsWidget.cpp

```diff
@@ -57,22 +57,26 @@ makeGradientCombo()
   int index = 0;
   for (auto& gspec : getBuiltInGradients()) {
     QLinearGradient gradient;
-    gradient.setStops(colormapToGradient(gspec.second));
+    gradient.setStops(colormapToGradient(gspec.m_stops));
     gradient.setStart(0., 0.);     // top left
     gradient.setFinalStop(1., 0.); // bottom right
     gradient.setCoordinateMode(QGradient::ObjectMode);
 
+    QString itemText;
     QBrush brush(gradient);
-    brush.setStyle(Qt::LinearGradientPattern);
-    cb->addItem("", QVariant(gspec.first.c_str()));
-    cb->setItemData(index, QVariant(gspec.first.c_str()), Qt::ToolTipRole);
+    if (gspec.m_name == "Labels") {
+      // special case for Labels
+      brush.setColor(QColor(255, 255, 255));
+      brush.setStyle(Qt::SolidPattern);
+      itemText = gspec.m_name.c_str();
+    } else {
+      brush.setStyle(Qt::LinearGradientPattern);
+    }
+    cb->addItem(itemText, QVariant(gspec.m_name.c_str()));
+    cb->setItemData(index, QVariant(gspec.m_name.c_str()), Qt::ToolTipRole);

```

Comment:
the above code is just here to set up the dropdown box for the UI, given a list of color gradients.